### PR TITLE
add thumbnail image

### PR DESCRIPTION
**Pull request by festinalente91**
- added get thumbnail image url

### CODE REVIEW

File: src/server/aws-s3.js [hunk 1]
```diff
@@ -63,53 +61,29 @@ export class S3Connector {
...
-      cb(null, urlList);
-    }).catch((err) => {
-      cb(err);
-    });
+    return urlList;
+  }
+  getPrefixedImageUrl(key, prefix) {
+    return this.getImageUrl(`${prefix}-${key}`);
+  }
+  getPrefixedImageurls(keys, prefix) {
```
Comment (ovekyc) [inline src/server/aws-s3.js]: Urls

File: src/server/images/controllers.js [hunk 2]
```diff
@@ -29,16 +29,7 @@ export default {
...
-          const conn = new S3Connector();
-          conn.getImageUrls(keys, (err, urls) => {
-            if (err) {
-              reject();
-              return;
-            }
-            resolve(urls);
-          });
-        });
+        return new S3Connector().getImageUrls(keys);
```
Comment (ovekyc) [inline src/server/images/controllers.js]: I think we can remove this 'then' block and just use 
urls = new S3Connector().getImgaeUrls(keys)

File: src/server/images/controllers.js [hunk 3]
```diff
@@ -104,11 +90,7 @@ export default {
       });
     })
     .then(() => {
-      return new Promise((resolve, reject) => {
-        s3.getImageUrl(key, (err, url)=>{
-          return (err) ? reject(err) : resolve(url);
-        });
-      });
+      return s3.getImageUrl(key);
```
Comment (ovekyc) [inline src/server/images/controllers.js]: same with line 32

File: src/server/images/controllers.js (hunk 2 above)
Comment (festinalente91) [reply src/server/images/controllers.js]: got a point!

File: src/server/images/controllers.js (hunk 3 above)
Comment (festinalente91) [reply src/server/images/controllers.js]: right

**Comment by ovekyc** [comment]
LGTM

**Comment by HewonJeong** [comment]
this.addComment('LGTM'')

**Comment by hewonjeong** [comment]
lgtm

**Pull request merged**